template/base/_pnpm-workspace.yaml

```diff
@@ -1,4 +1,9 @@
 # managed by skuba
+minimumReleaseAge: 4320
+minimumReleaseAgeExclude:
+  - '@seek/*'
+  - '*skuba*'
+  - '*seek*'
```

Comment:
Is it concerning to exempt package patterns that we don't control (i.e. other than `@seek/*`)?

I'm thinking: not great, but probably not a dealbreaker. `skuba` isn't going to appear in package names organically, so it would require a targeted attack and two steps to exploit (compromise a package `left-pad` to add a benign dependency on a `bad-skuba@^1.0.0` package then wait for 3 days for `left-pad` to be bumped, publish `evil-skuba@1.0.1` with badness included then wait for the next lock file maintenance). `seek` may be more likely but again would likely require a targeted attack.

Not sure I like the alternative of tediously listing out the exclusions, unless we keep it to a pretty small subset (e.g. it's probably fine if `seek-koala` is not updated immediately).

Reply:
Yeah thinking through it more, the motivation was to keep the exclusion list short while not getting in the way of development. But if we keep the ability to overload then that provides an escape hatch if required, and makes the default less greedy. 🤔